Apple Pie/Apple Pie/ViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ViewController.swift
+//  Apple Pie
+//
+//  Created by Student03 on 2019-04-15.
+//  Copyright © 2019 Student03. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var listOfWords = ["canada", "tree", "video", "hello", "apple", "hockey", "rock", "orange", "soccer", "computer", "game", "phone", "pie"]
+    
+    let incorrectMovesAllowed = 7
+    
+    var totalWins = 0 {
+        didSet {
+            newRound()
+        }
+    }
+    
+    var totalLosses = 0 {
+        didSet {
+            newRound()
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        newRound()
+        // Do any additional setup after loading the view.
+    }
+    
+    var currentGame: Game!
+    
+    func newRound() {
+        if !listOfWords.isEmpty {
+            let newWord = listOfWords.removeFirst()
+            currentGame = Game(word: newWord, incorrectMovesRemaining: incorrectMovesAllowed, guessedLetters: [])
+            enableLetterButtons(true)
+            updateUI()
+        } else {
+            enableLetterButtons(false)
+        }
+    }
+    
+    func enableLetterButtons (_ enable: Bool) {
+        for button in letterButtons {
+            button.isEnabled = enable
+        }
+    }
+    
+    func updateUI(){
+        var letters = [String]()
+        for letter in currentGame.formattedWord {
+            letters.append(String(letter))
+        }
+        let wordWithSpacing = letters.joined(separator: " ")
+        correctWordLabel.text = wordWithSpacing
+        scoreLabel.text = "Wins: \(totalWins), Losses: \(totalLosses)"
+        treeImageView.image = UIImage(named: "Tree \(currentGame.incorrectMovesRemaining)")
+        }
+    
+    @IBOutlet weak var treeImageView: UIImageView!
```

Comment:
Try to arrange your code in the future and and put those declaration at the top of the class. It's a convention for better readability